### ISSUE TITLE
Log level being overwritten

### ISSUE THREAD

**Issue by nr-goby** [baselime/edge-logger#7]
Hello, 

I'm getting a warning when bundling my worker with the Baselime logger:

    ▲ [WARNING] Duplicate key "level" in object literal [duplicate-object-key]

    [redacted].mjs:1927:116:
      1927 │ ...p: Date.now(), requestId: this.requestId, ...r, level: void 0 };
           ╵                                                    ~~~~~

      The original key "level" is here:

    [redacted].mjs:1927:26:
      1927 │     let a = { message: e, level: r?.level || t, traceId: o, time...

When I ran the debugger, I noticed the log level was being overwritten when the log was being declared within the _log function. (Line 108 index.ts)

```typescript
const log: BaselimeLog = {
	message,
	level: data?.level || level,
	traceId,
	timestamp: Date.now(),
	requestId: this.requestId,
	...{ ...data, level: undefined },
},
``` 

This has the side effect of error logs not showing up in the Baselime console as errors (which is how I found this behavior). When I edited the module locally to remove the re declaration of level, the log level was then working as expected and the wrangler warning disappeared from my console.

Is there any reason for level being set to undefined? Would removing that be an issue to any particular use case?

Essentially, my code looked like this:

```typescript
const log: BaselimeLog = {
	message,
	level: data?.level || level,
	traceId,
	timestamp: Date.now(),
	requestId: this.requestId,
	...{ ...data },
},
```

**Comment by treeder** [comment]
I ran into this too, and I applied the exact same changes as you @nr-goby to get it to work. It erases the level field basically.

**Comment by nr-goby** [comment]
@treeder I just downgraded to 0.2.1 and that has solved the issue for me as well.

**Comment by Ankcorn** [comment]
Thanks @treeder and @nr-goby, this is fixed in 0.3.1

https://github.com/baselime/edge-logger/releases/tag/v0.3.1
